### DISCUSSION TITLE
Account Suspended - Hacked

### DISCUSSION THREAD

**Discussion by illidanstoramge333** [community/community#147253] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Greetings,

I just wanted to ask for help regarding the suspension of my GitHub account github.com/talpiie 

Yesterday I was just doing work and when i tried to push my code into my repository — git notified me that my account was suspended.
So I went to support and tried to send a ticket, but when you send a ticket you also have to login so I cannot send a ticket because I cannot login since my account is suspended.

Then later on I noticed on my Visual Studio that I have to re-enter my credentials on my Microsoft account, upon trying to login and after clicking continue while choosing my email — it redirected me to get a code from an email that I don't recognize then my email was changed and I'm no longer able to login using my Microsoft email since it doesn't exist.

Since that Microsoft account was bound to my github, was that the reason why I got suspended? Anyone encountered a similar issue? 

I'm currently working for a software development company and my work is getting delayed since I can't push my files, and I don't know if i should create a new one to use because thats TOS.

I already sent a support ticket yesterday. 

Thanks for anyone who will reply

**Comment by MatinGhanbari** [comment]
Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)

**Comment by OmarrSakr** [comment]
Hi @illidanstoramge333,

I understand how frustrating this situation must be, especially with the disruption it’s causing to your work.

Based on the details you’ve provided, it seems like your account was either compromised or potentially targeted by a malicious actor, which may have led to the suspension. The change in your Microsoft email linked to your GitHub account could indeed be the cause of this, as it may have triggered GitHub’s security measures to protect your account.

Here are a few steps you can take:

Contact GitHub Support: Since you can't access your account, try reaching out to GitHub’s support team via a different account (if possible) or through their [contact page](https://support.github.com/contact). Explain the situation in detail, including the fact that your Microsoft account was hacked, and provide any relevant information to help them validate your identity.

Account Recovery: You may need to go through GitHub’s account recovery process. If your email was changed, GitHub support can help you verify your identity and regain access.

Check Security: Once your account is recovered, make sure to enable two-factor authentication (2FA) to add an extra layer of security. This is especially important since your account was compromised.

Backup Your Work: For your ongoing work, it’s crucial to find a way to keep your progress secure. You can try creating a new account temporarily (if GitHub allows you) to push your code while waiting for your original account to be restored.

As for whether creating a new account violates the GitHub TOS, it’s best to confirm directly with GitHub’s support team to ensure you’re not violating any rules while awaiting the resolution of your case.

Hopefully, you’ll regain access soon! Feel free to follow up if you need more assistance, and don’t hesitate to reach out to GitHub’s support for more specific guidance.

And if this helps, I’d appreciate it if you could mark it as the answer. 😊
